themes/simple/css/admidio.css

```diff
@@ -609,3 +609,18 @@ tr.admidio-event-highlight {
 #plugin-login-form .btn {
     margin-bottom: 1rem;
 }
+
+
+/***********************************/
+/* Changelog configuration  */
+/***********************************/
+
+div.changelog-tableselect-header {
+    font-weight: bold;
+}
+ul.changelog-tableselect-list {
+    list-style-type: none;
```

Comment:
Instead of setting this explicitly in the global css file you can set the class `list-unstyled` in the template at that place.

Reply:
Hmm, list-unstyled not only hides the bullet, but it also removes the indent and it adds vertical padding... 

![image](https://github.com/user-attachments/assets/5f9053b9-f6f5-4ab1-8cfd-a70dfa76f3a1)
![image](https://github.com/user-attachments/assets/3612965c-e02b-4821-bdb9-31aa5ee401f8)

Reply:
Ok, you don't like that style? We can leave it the way you implement it.